packages/garbo/src/familiar/freeFightFamiliar.ts

```diff
@@ -91,6 +94,16 @@ export function menu(options: MenuOptions = {}): GeneralFamiliar[] {
         limit: "experience",
       });
     }
+
+    if (mode === "target") {
```

Comment:
needs to check if you have snapper right?

Reply:
sorry also i made a few small tweaks from the browser and a file need to be formatted which is why the linter is complaining that's my bad

Reply:
I don't think so? If we need a filter, I'd rather add a universal filter.

Reply:
This does need to check if you have snapper

Reply:
And the check _should_ take place here